### PR TITLE
fix ubiquiti

### PR DESCRIPTION
**Pull request by lzammit**
# Description
As its name says, it renames all the Ubiquity naming to Ubiquiti

# Impacts
Switches / doc

# Delete branch after merge
YES

### CODE REVIEW

**Comment by louismunro** [comment]
I find the title to be somewhat ambiguous.
Could you elaborate?

**Comment by whitx** [comment]
* remove lib file named ubiquity
* add a UPGRADE notes/script to change in your conf/switch.conf

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -2854,8 +2854,8 @@ Now you will need to restart the pfqueue and the pfdetect services.
 
 Make sure that you have your pipe file otherwise the process won't start.
 
-StreamScan example:
--------------------
+StreamScan CDS example:
```
Comment (whitx) [inline docs/PacketFence_Administration_Guide.asciidoc:2857]: Why are those changes here?
